#goal definition
string boxId
bool planOnly
---
#result definition
moveit_msgs/MoveItErrorCodes error_code
---
#feedback
int32 status